partials/intro.html

```diff
@@ -1,7 +1,8 @@
 <div class="view-content intro-content container-fluid">
 	<div class="row">
+		<img src="img/curvy-bowl-dark-bg.png" id="intro-graphic">
 		<div class="primary-tagline col-xs-12 col-sm-offset-2 col-sm-8">
-			Freedom can be closer than you think
+			Reach freedom sooner
```

Comment:
Let's talk, but I'm liking 'Reach freedom sooner' for right now since it's more directly pertaining to what we do. 'Free yourself' sounds a bit dramatic and undescriptive for what we're doing.